src/api/index.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios"
+
+const API = axios.create({ 
+    baseURL: "http://localhost:5000/api/v1", 
```

Comment:
can we have this in the env file so that we may modify this link later on in production